wrappers/golang/internal/generator/poseidon/templates/poseidon_test.go.tmpl

```diff
@@ -1,59 +1 @@
 package tests
```

Comment:
We can remove this file if there are no tests in it. Also will need to remove the generation of it from the poseidon/generate.go